README.md

```diff
@@ -1,2 +1,146 @@
+<p align="justify">
 # Star-Wars-BB9
 Srishti 2020
+
+<p align="center">
+  <img width="450" height="550" src="https://github.com/Kritin02/Star-Wars-BB9/blob/master/Images%20and%20Videos/Images/Interior.jpeg?raw=true">
+</p>
+
+## Abstract
+The droid should replicate BB-8 from the Star Wars franchise complete with a rotating body but a stationary head.The movement of the droid can be controlled by a companion app. The body can rotate freely and can be moved in all direction freely using the app.The head is established by using strong Neodymium magnets and ball casters.* 
```

Comment:
@Kritin02 
Remove asterisk sign from the end of the line and use p tag separately for each paragraph